<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>425</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>投屏 by 最强路痴</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="screenshotLabel">
        <property name="minimumSize">
         <size>
          <width>280</width>
          <height>171</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,2,0,2,1">
        <item>
         <widget class="QLabel" name="hostLabel">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="hostLineEdit"/>
        </item>
        <item>
         <widget class="QLabel" name="portLabel">
          <property name="text">
           <string>端口</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portLineEdit"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>宽</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="img_w_lineEdit">
            <property name="text">
             <string>240</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>高</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="img_h_lineEdit">
            <property name="text">
             <string>240</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="connectBtn">
          <property name="text">
           <string>开启投屏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="disconnectBtn">
          <property name="text">
           <string>关闭投屏</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="selectregionBtn">
          <property name="text">
           <string>选择显示范围</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="fps_Label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>帧率：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>529</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
